### PR TITLE
Refine Search Interface

### PR DESCRIPTION
**Pull request by amCap1712**
Add animation to result items.
Fix Search Suggestions.

### CODE REVIEW

**Comment by Miguel-Herrero** [comment]
Cool animation! Have you noticed that it only animated the first time? If I search a second artist, results are no longer animated.


animation

**Comment by amCap1712** [comment]
No, i didn't. I'll fix it soon.

**Comment by amCap1712** [comment]
The latest commit seems to have fixed the animations. Also a "No Results Found" message is added.

File: app/src/main/java/org/metabrainz/mobile/activity/SearchActivity.java [hunk 1]
```diff
@@ -189,13 +166,17 @@ private void chooseAdapter() {
 
     private void doSearch(String query) {
         saveSearchSuggestion(query);
+        progressBar.setVisibility(View.VISIBLE);
+        adapter.resetAnimation();
```
Comment (amCap1712) [inline app/src/main/java/org/metabrainz/mobile/activity/SearchActivity.java]: Reset animation status before each query.

File: app/src/main/java/org/metabrainz/mobile/adapter/list/SearchAdapter.java [hunk 2]
```diff
@@ -25,4 +25,8 @@ protected void setAnimation(View viewToAnimate, int position) {
             lastPosition = position;
         }
     }
+
+    public void resetAnimation() {
```
Comment (amCap1712) [inline app/src/main/java/org/metabrainz/mobile/adapter/list/SearchAdapter.java]: To reset animation index.

**Pull request merged**